harvester_e2e_tests/integration/test_vm_functions.py

```diff
@@ -1,15 +1,20 @@
-from time import sleep
+# noqa: E501
+import json
+from datetime import datetime, timedelta
 from pathlib import Path
 from tempfile import NamedTemporaryFile
-from datetime import datetime, timedelta
+from time import sleep
```

Comment:
It looks we Import json but modify other import's order, I think we can just do `import json` in the first or the last would be enough.

And I have no idea why we should skip `E501` in here.

Reply:
yeah, that's a great catch, I've gone ahead and removed the ignoring of `E501` - I'm not sure if that was there for some reason prior, perhaps I added it when testing something :sweat_smile:  - I've also updated the placement of the `import json` within the file :smile: